tools/testbench/topology.c

```diff
@@ -734,6 +734,9 @@ int parse_topology(struct sof *sof, struct shared_lib_table *library_table,
 			temp_comp_list = (struct comp_info *)
 					 realloc(temp_comp_list, size);
 
+			for (i = (num_comps - hdr->count); i < num_comps; i++)
```

Comment:
I'm wondering if maybe we should just memset the whole new realloced section to 0 to be safe

Reply:
Won't it be a overkill? Only ```name``` member of  ```struct comp_info``` is a pointer.

Reply:
Let's see what the other reviewers think, but initializing memory is usually a good practice

Reply:
@cujomalainey ack
@ymdatta fwiw, probably worth running fuzzer a few times in ```valgrind --tool=memcheck''', this will root out any other issues.

Reply:
@lgirdwood: Related to this structure, in valgrind, there were no errors for other structure members. (There were other memory leaks, i'll send the patches to those separately, to make the change a single logic one, will that work?)